.github/workflows/deploy-dev.yml

```diff
@@ -24,3 +24,20 @@ jobs:
     with:
       SKIP_DRAFT_RELEASE: ${{ github.ref_name != 'master' }}
     secrets: inherit
+  setup-dekorator:
```

Comment:
Hva er det denne actionen gjør? Ser ut som om den bare kjører npm install, men blir det brukt noe sted?

Reply:
Det er slik det står det skal settes opp i dok'en for dekoratøren så derfor tok jeg det med for sikkerhets skyld, men tror egentlig ikke den trengs. Jeg skal teste litt mer rundt det.